packages/version/src/__tests__/update-lockfile-version.spec.js

```diff
@@ -72,7 +78,36 @@ describe('npm modern lock file', () => {
       await saveUpdatedLockJsonFile(lockFileOutput.path, lockFileOutput.json);
     }
 
-    expect(Array.from(loadJsonFile.registry.keys())).toStrictEqual(["/packages/package-1", "/packages/package-2", "/"]);
+    expect(Array.from(loadJsonFile.registry.keys())).toStrictEqual([
+      '/packages/package-1',
+      '/packages/package-2',
+      '/'
+    ]);
     expect(fs.readJSONSync(rootLockFilePath)).toMatchSnapshot();
   });
-});
\ No newline at end of file
+});
+
+describe('pnpm lock file', () => {
+  test('updateModernLockfileVersion v2 on pnpm lock file', async () => {
```

Comment:
I don't think "updateModernLockfileVersion v2" has anything to do with pnpm. I added this "modern" name mainly for npm lock file v2 (npm >=7) which is quite different from npm <=6). So anyway, since you're adding pnpm, we probably don't need "modern" and "v2" in this test description, it's probably better to rename this.